src/stories/4-elements/wrapper/wrapper.stories.tsx

```diff
@@ -34,11 +34,11 @@ export default {
 			description: "Actual HTML element type to render the component as",
 			table: {
 				type: {
-					summary: "div|span|section|header|footer|h1|h2|h3|h4|h5|h6|p",
```

Comment:
understand that the types are removed to prevent conflicting with the new `Text` field, however I think these types are still needed. can we prepend all the field types in `Text` with `text-`?

let's also try to standardise the fieldType naming convention to be kebab-case, e.g. `text-dbody`, `text-h1`, `text-xsmall`

I also noticed the `Various Elements` story is broken, but I think this change will fix the issue